syntax = "proto3";
package smartbox_msgs;

service CameraController {
	rpc get_current_image (CameraImageRequest) returns (CameraImageResponse) {}
}

message CameraImageRequest {
	string message = 1;
}

message CameraImageResponse {
	bytes data = 1;
	uint32 width = 2;
	uint32 height = 3;
}
